ghost/email-service/lib/email-renderer.js

```diff
@@ -294,7 +293,7 @@ class EmailRenderer {
         if (options.clickTrackingEnabled) {
             html = await this.#linkReplacer.replace(html, async (url) => {
                 // Decode any escaped entities in the url
-                url = new URL(entities.decode(url.toString()));
+                url = new URL(url.toString());
```

Comment:
This line i guess is redundant now, as even if we remove this `url` is already of type `URL` 😄